app/controllers/checkout_controller.rb

```diff
@@ -9,9 +9,11 @@ def show
   end
 
   def create
+    require "pry-nav"; binding.pry
     if order_params[:status].downcase === "completed"
       @current_order.update_order(session[:order], order_params)
       if @current_order.save_order(@current_user)
+        require "pry-nav"; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.